src/Conversion/KrnlToLLVM/KrnlEntryPoint.cpp

```diff
@@ -47,17 +47,24 @@ class KrnlEntryPointOpLowering : public OpRewritePattern<KrnlEntryPointOp> {
   SmallVectorImpl<LLVM::GlobalOp> &entryGlobalOps;
   SmallVectorImpl<LLVM::GlobalOp> &inSigGlobalOps;
   SmallVectorImpl<LLVM::GlobalOp> &outSigGlobalOps;
+  std::map<std::string, SmallVector<MemRefType, 4>> &inputMemRefTypes;
+  std::map<std::string, SmallVector<MemRefType, 4>> &outputMemRefTypes;
```

Comment:
Changes in `KrnlEntryPoint.cpp` is to handle unsigned integer outputs. LLVM IR only uses signless integers, so we have to store the original MLIR type (e.g. ui8) so that we can construct output OMTensors with correct types.